pandas/tests/indexes/period/test_astype.py

```diff
@@ -164,3 +166,24 @@ def test_period_astype_to_timestamp(self):
         res = pi.astype("datetime64[ns, US/Eastern]", how="end")
         tm.assert_index_equal(res, exp)
         assert res.freq == exp.freq
+
+    def test_astype_tz_conversion_roundtrip(self):
```

Comment:
this is in the wrong test file, should be for datetimes

Reply:
probably tests/frame/methods/test_astype.py

Reply:
moved.